src/Codeception/Coverage/Subscriber/Printer.php

```diff
@@ -146,6 +150,12 @@ protected function printCrap4j()
         $writer->process(self::$coverage, $this->absolutePath($this->options['coverage-crap4j']));
     }
 
+    protected function printCobertura()
+    {
+        $writer = new \SebastianBergmann\CodeCoverage\Report\Cobertura;
```

Comment:
Please check if this class is defined before using it and throw exception that php-code-coverage >=9.2.0 is required.